docs/user-guide/connectors/index.md

```diff
@@ -5,6 +5,10 @@ The current supported connectors are:
 * `jsdom`: Your website will be loaded using [`jsdom`][jsdom].
 * `chrome`: Your website will be loaded using Chrome and the Chrome
   Debugging Protocol. This is one of the `remote-debugging-connector`s
+* `edge`: Your website will be loaded using Edge via the [`edge-diagnostics-adapter][eda].
+  You will need to run Windows 10 Creators Update or better to use it.
```

Comment:
> or better

or later

Reply:
Technically the Falls Creator Update is the best windows 10 version yet 😛